tests/edge_cases.luau

```diff
@@ -36,13 +36,25 @@ return function(test: typeof(frktest.test))
 			check.equal(targetPath, "pandoc")
 
 			local bin = zip:extract(entry, { type = "text", followSymlinks = true }) :: buffer
-			local expectedBin =
-				process.spawn("unzip", { "-p", "tests/data/pandoc_soft_links.zip", "pandoc-3.2-arm64/bin/pandoc" })
-			check.is_true(expectedBin.ok)
+
+			-- Fix pipe buffer overload by writing data into temp file and reading the temp file
+			local tmpOut = "/tmp/unzip_meta_"
+				.. tostring(os.time())
+				.. "_"
+				.. tostring(math.random(1, 1_000_000_000))
+				.. ".txt"
```

Comment:
We need to look into a better temp file generation to prevent collision. I wonder if there is a library present for this in Luau, could be handy.